Polyhedron/demo/Polyhedron/Plugins/Mesh_3/C3t3_io_plugin.cpp

```diff
@@ -448,7 +449,55 @@ try_load_other_binary_format(std::istream& is, C3t3& c3t3)
     Fake_c3t3::Triangulation,
     C3t3::Triangulation,
     Update_vertex,
-    Update_cell>(is, c3t3.triangulation());
+    Update_cell>(is, c3t3.triangulation(),
+                 Update_vertex(),
+                 Update_cell()
+#ifdef CGAL_CXX11
+                 ,
+                 /* call_back: a C++11 lambda */
+                 [](std::istream& is,
+                    const C3t3::Triangulation::Triangulation_data_structure& tds,
+                    const char* text = 0)
+                 {
+                   static CGAL::Real_timer timer;
+                   static double last_time = 0;
+
+                   if(!timer.is_running()) {
+                     std::cerr << "Start loading...\n";
+                     timer.start();
+                   }
+                   if(text != 0) { std::cerr << text; return; }
+                   const double current_time = timer.time();
+                   if(current_time > last_time + 1.) {
```

Comment:
I agree, in general, because `time()` can make a syscall. But in my case the loop itself is just a lot of syscalls, that read the file. I do not think that can slow down. ... 

-> to bench.